%s/Micronaut Framework/the Micronaut framework/

sdelamo opened the pull request.

sdelamo committed:
%s/Micronaut Framework/the Micronaut framework/

kroegerk committed:
Capitalization update

In this case, "framework" is part of a title, so it remains capitalized.

kroegerk committed:
Capitalization update

Framework is part of a title

kroegerk committed:
Capitalization

Framework is part of a title in this instance

kroegerk committed:
Added hyphens

Step-by-step is hyphenated when used as an adjective

kroegerk committed:
Micronaut already used as an adjective

In this case, Micronaut modifies "caching annotations," so it is being used correctly as an adjective. The only time you need to add "framework" is in cases where Micronaut is being used as a noun.

kroegerk committed:
Capitalization update

"Error" is not a proper noun, so it does not need to be capitalized in a sentence.

kroegerk committed:
Capitalization and punctuation update

Framework is correctly capitalized because this is a title; "Event-Driven Applications" is also part of a title, so it needs to be capitalized as well. Also because "event-driven" is being used as a compound adjective, it needs to be hyphenated.

kroegerk committed:
Add "the" before Micronaut Framework

In previous guides I've proofread, <h2> level headings were in title case and <h3> level headings were in sentence case. If that is a style convention you wish to keep consistent, this <h2> would have "framework" capitalized. I also added "the" before "Micronaut." It also just occurred to me as I was typing this note that all headings in the guides are transformed to uppercase text via the CSS, so it actually doesn't matter how headings are capitalized in the Markdown. *facepalm

kroegerk committed:
Capitalization update

Framework is capitalized when used in the title

kroegerk committed:
Title in title case

Capitalized the words that needed capitalizing in the title

kroegerk committed:
Title case update

Capitalized the words in the title that needed to be capitalized

kroegerk committed:
Capitalization update

Put guide title in title case

kroegerk committed:
Capitalization update

Put guide title in title case

kroegerk committed:
Capitalization update

Multitenancy and microservices are not proper nouns, so I put them in lowercase.

kroegerk committed:
Title case

Put the guide title in title case

kroegerk committed:
Capitalization update

title case

kroegerk committed:
Capitalization update

title case

kroegerk committed:
Capitalization update

Title case for <h2>

kroegerk reviewed (approved):
I submitted separate pull requests for files I proofread completely. All basic capitalization and "framework-addition" adjustments have been made and can be merged.

kroegerk committed:
Proofreading (#550)

Minor changes. 

In line 5, there's no need to add "framework" because "Micronaut" is being used as a modifier for "capabilities." 

It isn't clear whether you'd like "job" to be treated as a proper noun; I capitalized it for consistency.

kroegerk committed:
Proofreading (#549)

Minor changes. Also, on Line 80, should that read, "The previous service **returns** a 'List<book>'"?

kroegerk committed:
Proofreading (#548)

Minor changes

kroegerk committed:
Proofreading (#547)

Minor changes.

kroegerk committed:
Proofreading (#546)

Minor corrections

kroegerk committed:
Proofreading (#545)

Several minor corrections. Also, on line 201, it looks like there may be a typo. It says, "The previous request generates a traces composed by 9 spans." "A traces" doesn't sound right to me.

sdelamo committed:
Merge branch 'master' into the-micronaut-framework
